src/foam/dao/history/HistoryRecordServiceServer.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao.history',
+  name: 'HistoryRecordServiceServer',
```

Comment:
These are usually named as "ServerHistoryRecordService"

Reply:
I prefer this.